### PR TITLE
Design update

### PR DESCRIPTION
**Pull request by brunosj**

### CODE REVIEW

File: src/components/timetable/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const Timetable = () => {
   const { weekdaysThisAndNextWeek } = useAllShows()
 
   return (
-    <Section title="timetable">
+    <Section title="timetable" className="bg-thfBlue">
```
Comment (nextlevelshit) [inline src/components/timetable/index.js:16]: `thfBlue` :sunglasses: cooool

**Review by nextlevelshit — COMMENTED** [review]
Is this already online? I am curious for the `slick slider`. Good luck!

**Comment by brunosj** [comment]
Hey Michał! The slider is not yet implemented. Still need to figure some things out..!

File: src/components/timetable/index.js (hunk 1 above)
Comment (brunosj) [reply src/components/timetable/index.js:16]: Yeeessss

**Pull request merged**